src/position/position.js

```diff
@@ -436,8 +436,8 @@ angular.module('ui.bootstrap.position', [])
         targetElem = this.getRawNode(targetElem);
 
         // need to read from prop to support tests.
-        var targetWidth = angular.isDefined(targetElem.offsetWidth) ? targetElem.offsetWidth : targetElem.prop('offsetWidth');
-        var targetHeight = angular.isDefined(targetElem.offsetHeight) ? targetElem.offsetHeight : targetElem.prop('offsetHeight');
+        var targetWidth = angular.element(targetElem).outerWidth();
+        var targetHeight = angular.element(targetElem).outerHeight();
```

Comment:
Those are jQuery methods (outerWidth and outerHeight) and not native element or jQlite methods.  The note above the line of code removed says the prop lookups are there for a reason - to support the tests, so I'm not sure why they are removed.  The position test creates a mock element [here](https://github.com/angular-ui/bootstrap/blob/master/src/position/test/position.spec.js#L6) with those values as props.